webapp/src/extensionsBrowser.tsx

```diff
@@ -448,18 +448,24 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
                         }
                         {displayMode == ExtensionView.Tabbed &&
                             <div className="tab-header">
-                                <Button
-                                    title={lf("Recommended")}
-                                    label={lf("Recommended")}
-                                    onClick={() => { setCurrentTab(TabState.Recommended) }}
-                                    className={currentTab == TabState.Recommended ? "selected" : ""}
-                                />
-                                <Button
-                                    title={lf("In Development")}
-                                    label={lf("In Development")}
-                                    onClick={() => { setCurrentTab(TabState.InDevelopment) }}
-                                    className={currentTab == TabState.InDevelopment ? "selected" : ""}
-                                />
+                                {!!extensionsInDevelopment.length &&
+                                    <>
+                                        <Button
+                                            title={lf("Recommended Extensions")}
+                                            label={lf("Recommended")}
+                                            onClick={() => { setCurrentTab(TabState.Recommended) }}
+                                            className={currentTab == TabState.Recommended ? "selected" : ""}
+                                        />
+                                        <Button
+                                            title={lf("Local GitHub Projects")}
+                                            label={lf("In Development")}
```

Comment:
I also second the comment of Mark. "In Development" is somewhat ambiguous. Why did we change it from Local?

(this is mostly hidden from all users, just curious).

Reply:
Shoot, merged the PR before I saw this comment. I'll make a change in my next PR if we come to a conclusion here.

Reply:
It's what we had when the new extension page was being designed. I don't mind changing it from "In Development" to "Local" if people feel that would be clearer/easier to understand.